packages/experimental/Shadow/src/shadowStyle.ts

```diff
@@ -2,6 +2,10 @@ import { memoize } from '@fluentui-react-native/framework';
 import { ShadowToken } from '@fluentui-react-native/theme-types';
 import { ColorValue } from 'react-native';
 
+// The blur property from the token is not the same as the shadow radius value,
+// it needs to be divided by 2 to achieve the same effect.
+const shadowBlurAdjustment = 0.5;
+
```

Comment:
I wonder 2 things:

1) Should we do this "transform" at the token pipeline level before it ever gets to FURN? My initial feeling is no, since the `shadowToken` type is not RN Specific at the moment and more like a designer value. `shadowStyle.ts` is doing the transform to RN types for us, so it feels appropriate here. But then.. should we move the whole `shadowStyle` function to the token pipeline then?


2) Should we restrict this per platform?

It seems this is an iOS /macOS specific issue.. If we can get a repro on win32 that would be nice. Till then could we only do the adjustment if we're macOS or iOS?

Reply:
1. I briefly talked to Hunter about this, I think the value in the FUA token pipeline is the same as the Figma value now (at some point it was the adjusted value, but got switched back). Not really sure what you mean with moving the whole shadowStyle function to the token pipeline?
2. Yes possibly, I wasn't sure if win32 would require the same fix or not. I actually thought it was more likely that it would require the same fix, otherwise right now adding a shadow to RNmacOS vs. RNwin32 would look different? I noted it down as something that should be checked on win32

Reply:
Update after a lot of investigation/discussion offline- left shadowStyle function in FURN, and restricted to macOS/iOS.